integration_tests/sdk/test-credentials-example.yml

```diff
@@ -25,6 +25,12 @@ data:
     # (Alternative) Config File Credentials
     # config_file_path:
     # config_file_profile:
+    # Whether to configure this integration as the metadata store too.
+    # use_as_storage:
+  test_mongo_db:
+    type: MongoDB
+    auth_uri: # the URI from cluster -> connection -> connect to application.
```

Comment:
Not sure what the arrows mean here :P Can you just edit it to just use words?

Reply:
This is a click sequence in UI. Adding some clarifications here